README.md

```diff
@@ -21,59 +21,144 @@ locale. The "core" `Faker\Generator` contains a set of predefined of methods; `f
 `address()`, `companyName()` etc, which is just syntactic sugar for calling the
 extensions.
 
-Here is how to use an extension:
+### Registering an extension
+
+We use PSR-11 to configure extensions:
 
 ```php
-$faker->ext(Foo::class)->bar();
+<?php
+use Example\Person;
+use Faker\ContainerBuilder;
+use Faker\Factory;
+use Psr\Container;
+
+$containerBuilder = new ContainerBuilder();
+
+$containerBuilder->register(Person::class);
+
+/** @var Container\ContainerInterface $container */
+$container = $containerBuilder->build();
+
+$faker = Factory::create(
+    null, 
```

Comment:
Not sure about this argument, what should it be?

Reply:
Oh, that is true.. It was a "provider", but we removed that.

This line should be removed.